### DISCUSSION TITLE
Fubar changed IP on VM and now dup ports

### DISCUSSION THREAD

**Discussion by Googlepuss** [pterodactyl/panel#5015] [Support] [answered]
I reserved a static for the VM (Ubuntu serv 22.4) running this app. Then noticed the node I created was mapped to old address. Changed that. Ran wings debug got ports already used error for 8080, 2022. Blew away config.yml deleted servers and node, rebuilt new node in same way. Will not pass debug still error on port 8080. “Wings” is the process that is using 8080 currently on machine. Am at a loss. Feels like I am missing something simple.

**Comment by Googlepuss** [comment, marked as answer]
Well. Sh!t. Reboot worked.

**Reply by trenutoo** [reply]
Running Wings command will execute the binary manually, while systemd would already have it running in the background unless stopped.
